src/client/cli/argparser.cpp

```diff
@@ -156,6 +156,18 @@ mp::ParseCode mp::ArgParser::commandParse(cmd::Command* command)
     if (!parsedOk)
     {
         cerr << qUtf8Printable(parser.errorText()) << '\n';
+
+        if (parser.errorText().contains("Unknown option"))
+        {
+            cerr << "\nYou need to separate the options to be passed to the command with \"--\", like this: \n";
+            QStringList suggestedCommand(arguments);
+            suggestedCommand.insert(3, "--");
```

Comment:
Getting the right command in all cases would be quite tricky. A few examples of difficulties: 
  * the input could require the `--` to be in a different place (e.g. `multipass -vvv exec ...`)
  * bad options might be mistakes on genuine multipass options (e.g. `multipass exec --helpp`)
  * there could be more than one problem (e.g. `multipass --nonsense exec ls -la`)
  * If arguments had special characters, when printing the amended command back, we'd have to at least quote those

For these reasons, I think it's best to just say that the right usage is `multipass exec <instance> -- <command> <arguments>`.